changelog.d/2308.bugfix

```diff
@@ -0,0 +1 @@
+Fix 'There are multiple DataStores active for the same file' crashes
```

Comment:
I am tempted to say that it's not necessary to add a file for the changelog, since the crash was not on the previous release.